aws/table_aws_config_conformance_pack.go

```diff
@@ -0,0 +1,172 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/configservice"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+func tableAwsConfigConformancePack(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_config_conformance_pack",
+		Description: "AWS Config Conformance Pack",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("name"),
+			ShouldIgnoreError: isNotFoundError([]string{"NoSuchConformancePackException"}),
+			Hydrate:           getConfigConformancePack,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listConfigConformancePacks,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
```

Comment:
Can you please add an `arn` column that is transformed from the `ConformancePackArn` field?